lib/rpc/config.ts

```diff
@@ -0,0 +1,19 @@
+export interface Config {
+  ERROR_PENALTY: number
+  HIGH_LATENCY_PENALTY: number
+  HEALTH_SCORE_FALLBACK_THRESHOLD: number
+  HEALTH_SCORE_RECOVER_THRESHOLD: number
+  MAX_LATENCY_ALLOWED_IN_MS: number
+  RECOVER_SCORE_PER_MS: number
+  RECOVER_EVALUATION_WAIT_PERIOD_IN_MS: number
+}
+
+export const DEFAULT_CONFIG: Config = {
+  ERROR_PENALTY: -50,
```

Comment:
might be worth commenting what these means; in other words, if we accumulate up to 3990 penalties in 5seconds then we fallback; I assume thats the case, and you can also explain the recovery part too